flutter_secure_storage_web/pubspec.yaml

```diff
@@ -13,6 +13,7 @@ dependencies:
   flutter_secure_storage_platform_interface: ^1.1.1
   flutter_web_plugins:
     sdk: flutter
+  web: '>=0.5.0 <1.0.0'
   js: ^0.6.3
```

Comment:
why is js still a dependency?

Reply:
Don't know why it's shown here. When I look at the file at my repo, it isn't. https://github.com/svenopdehipt/flutter_secure_storage/blob/develop/flutter_secure_storage_web/pubspec.yaml

Reply:
Just saw, that the file you reference is outdated. Please look at the changes now.

Reply:
Oh you changed it in the last commit: https://github.com/mogol/flutter_secure_storage/pull/698/commits/c9f2b62c1ab8831fc086c6563f0e154f49803f82#diff-dd1ec4717f1b8774ed50d5be49319e0a3e9976c07f16f133ac101a5f07066a3e (where you added it), where you removed it: https://github.com/mogol/flutter_secure_storage/pull/698/commits/dd4d5d0306468048b70c05b3008bf26565c27530

Reply:
I am resolving some merge conflicts due to the recent messy revert of some breaking changes, it won't be added when i merge this.